### PR TITLE
Feature/filtering pane

### PR DESCRIPTION
**Pull request by Jen-ni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,4 +33,7 @@ Wrappers and containers, that will help to build a solid base for your view.
 
 #### [Primitives](./src/primitives/README.md)
 Collection of low level styled components. Building blocks for composing all the more
-advanced layout compoents.
\ No newline at end of file
+advanced layout compoents.
```
Comment (bo-kh) [inline README.md]: typo, should be components

File: src/filtering-pane/filtering-pane.component.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    </FilteringPaneSection>
+  );
+}
+
+FilteringPane.propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string,
+  tabs: PropTypes.shape({
+    activeKey: PropTypes.number,
+    list: PropTypes.arrayOf(PropTypes.shape({})),
```
Comment (bo-kh) [inline src/filtering-pane/filtering-pane.component.jsx]: Shouldn't  it be immutable?

Comment (Jen-ni) [reply src/filtering-pane/filtering-pane.component.jsx]: If yes, PropType should be corrected to @opuscapita/react-responsive-navbar.

File: README.md (hunk 1 above)
Comment (Jen-ni) [reply README.md]: Fixed.

**Comment by Jen-ni** [comment]
Filtering pane moved to own repo

**Pull request closed without merging**